### PR TITLE
API swagger doc updates + names no conflict with OGC branch

### PR DESCRIPTION
**Pull request by fmigneault**
- more cornice-swagger definitions
- rename some methods to provider_processes to avoid conflicts with 'local' processes (OGC)
(only cherry-picked changes valid for this branch)
- swagger-ui is under `{twitcher}/doc` and it's json OpenAPI under `{twitcher}/json`
- update routes output=>result to match OGC standard
- adjust returned JSON to be HTTP Ok/Created/BadRequest responses as required
- method to return 'error' HTTP with basic information (code + description)

### CODE REVIEW

File: twitcher/wps_restapi/utils.py [hunk 1]
```diff
@@ -21,3 +29,46 @@ def jsonify(value):
...
+            return 'json'
+        return 'xml'
+    # version not specified as input, check accept headers
+    accepts = [accept[0] for accept in request.accept.parsed]
+    if 'application/xml' in accepts:
+        return 'xml'
+    return 'json'
+
+
+def httpError(http_error_class, **body):
```
Comment (dbyrns) [inline twitcher/wps_restapi/utils.py]: Test if you could just raise the HTTPException (json formatting should be handled by tween)

File: twitcher/wps_restapi/utils.py [hunk 2]
```diff
@@ -21,3 +29,46 @@ def jsonify(value):
     # other type
     else:
         return value
+
+
+def get_wps_output_format(request):
```
Comment (dbyrns) [inline twitcher/wps_restapi/utils.py]: Only needed for OGC usecases

**Review by dbyrns — CHANGES_REQUESTED** [review]

**Comment by fmigneault** [comment]
@dbyrns changes done

**Review by dbyrns — APPROVED** [review]

**Comment by fmigneault** [comment]
@dbyrns ready for merge

**Review by dbyrns — APPROVED** [review]

**Pull request merged**